common/test/acceptance/pages/lms/learner_profile.py

```diff
@@ -253,10 +253,6 @@ def remove_profile_image(self):
         self.wait_for_ajax()
 
         self.wait_for_element_visibility('.image-wrapper', "remove button is visible")
-        self.browser.execute_script('$(".u-field-remove-button").css("opacity",1);')
```

Comment:
I changed how the button is hidden and shown, so these tests aren't relevant anymore.